src/HelixMarkdownPreview.js

```diff
@@ -165,15 +165,15 @@ if (typeof window.HelixMarkdownPreview === 'undefined') {
       }
 
       /**
-       * Removes the preview window and resets the extension.
+       * Removes the preview window closes the popup and resets the extension.
        * @private
        */
       function removePopup() {
         if (popup) {
           delete popup.init;
           popup.close();
         }
-        chrome.extension.getViews({ type: 'tab' }).forEach((v) => {
+        chrome.extension.getViews({ type: 'popup' }).forEach((v) => {
```

Comment:
the preview window stays open when i close the browser tab with the markdown editor